### PR TITLE
Add docs on how to create users

### PR DESCRIPTION
**Pull request by alistaire47**
Just adding a bit to the README about user creation options

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -229,6 +230,14 @@ recorded values will not be recalculated if this value is changed. If you would
 like to change previous values, you would need to write a migration of the data
 to recalculate history.
 
+## Creating accounts
+
+Follow the login screen's "Sign Up" link, and use the registration key specified
+in the server configuration above.
+
+To create an additional account, when logged in, go to the gear menu / Users and
+use the "Add User" button to create a new account without the registration key.
```
Comment (jonkeane) [inline README.md]: Could we also add something like:

If you have set up conbench using an SSO provider, but you're looking to make a user name and password for API access, ...

File: README.md [hunk 2]
```diff
@@ -229,6 +230,14 @@ recorded values will not be recalculated if this value is changed. If you would
 like to change previous values, you would need to write a migration of the data
 to recalculate history.
 
+## Creating accounts
+
+Follow the login screen's "Sign Up" link, and use the registration key specified
+in the server configuration above.
```
Comment (jonkeane) [inline README.md]: Should we mention something here about "if you're a user of conbench, you might need to talk to your conbench administrator to get the registration key"?

File: README.md [hunk 3]
```diff
@@ -229,6 +230,14 @@ recorded values will not be recalculated if this value is changed. If you would
 like to change previous values, you would need to write a migration of the data
 to recalculate history.
 
+## Creating accounts
+
```
Comment (jonkeane) [inline README.md:234]: Could we also add in a bit here that describes how _by default_ conbench has open read access, so a user account is not required (even for API access).

But if you have a private conbench instance or you're attempting to write data to conbench, a user account is required.

**Review by jonkeane — CHANGES_REQUESTED** [review]
Thanks for this! A few comments | expansions that we should add.

File: README.md (hunk 1 above)
Comment (alistaire47) [reply README.md]: Tried again. Better?

File: README.md [hunk 4]
```diff
@@ -229,6 +230,23 @@ recorded values will not be recalculated if this value is changed. If you would
...
+By default, conbench has open read access, so a user account is not required to
+view results or read from the API. An account is required if the conbench instance
```
Comment (jonkeane) [inline README.md]: Maybe "is required only if" (sorry apparently I can't suggest from the mobile interface!)

**Review by jonkeane — APPROVED** [review]
Looks good. If we have docs already for setting up with SSO we might link them. If not, could you make an issue to add them (I don't want to bloat this PR, but we really should document that)

**Comment by alistaire47** [comment]
> If we have docs already for setting up with SSO we might link them. If not, could you make an issue to add them (I don't want to bloat this PR, but we really should document that)

https://github.com/conbench/conbench/issues/392

**Pull request merged**